src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BALL_NUMBER_MAX_VALUE = 100;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Ball getRandomBall() {
+        Color color = colorSupplier.getRandomColor();
```

Comment:
why this variable called just color while next one called randomBallNumber, please make names consistantly, decide to add that it is random ball something or not